.scripts/ci.yml

```
@@ -78,9 +78,9 @@ stages:
           vmImage: $[coalesce(variables['OSVmImage'], '')]
         steps:
           - template: build.yml
-          - script: npm run generate-swaggers
+          - script: cd $(Build.SourcesDirectory)/packages/autorest && npm run generate-swaggers
```

You can do this instead of cd
```suggestion
          - script: npm run generate-swaggers
          - workingDirectory: $(Build.SourcesDirectory)/packages/autorest
```